eva_submission/submission_qc_checks.py

```diff
@@ -268,6 +284,154 @@ def check_if_browsable_files_entered_correctly_in_db(self, vcf_files):
             expected files: {vcf_files}
             missing files: {missing_files if missing_files else 'None'}"""
 
+    def clustering_check_report(self, target_assembly):
+        clustering_log_file = glob.glob(f"{self.path_to_data_dir}/53_clustering/logs/{target_assembly}_clustering.log")
+        clustering_qc_log_file = glob.glob(
+            f"{self.path_to_data_dir}/53_clustering/logs/{target_assembly}_clustering_qc.log")
+
+        clustering_error = ""
+        if clustering_log_file:
+            if not self.check_if_job_completed_successfully(clustering_log_file[0], 'clustering'):
+                clustering_error += f"failed_job - {self.get_failed_job_name(clustering_log_file[0])}"
+                clustering_check_result = "FAIL"
+            else:
+                clustering_check_result = "PASS"
+        else:
+            clustering_error += f"Clustering Error : No clustering file found for {clustering_log_file[0]}"
```

Comment:
`clustering_log_file` is empty in this `else` clause, if you want to include the path to the file you _didn't_ find it should be extracted as a separate variable.  Same goes for other similar parts of the code.

Reply:
corrected in all the places